protocol/bc/outputid.go

```diff
@@ -0,0 +1,60 @@
+package bc
+
+import (
+	"database/sql/driver"
+	"io"
+
+	"chain/crypto/sha3pool"
+	"chain/encoding/blockchain"
+)
+
+// OutputID identifies previous transaction output in transaction inputs.
+type OutputID Hash
```

Comment:
Thanks, I didn't know about this feature. I've also tried to use `outid.Bytes()` instead of `tmp := outid.Hash; tmp[:]` when I need a byteslice. Is that the right way? 9042b59937f3f507bed9864049bacf8ba1724c60